common/source/docs/common-telemetry-port-setup.rst

```diff
@@ -45,6 +45,8 @@ MAVLink2 supports "signing" of packets which improves security and flexible leng
 
 As mentioned on the :ref:`Telemetry Logs page <common-mission-planner-telemetry-logs>` the rate messages are sent from the vehicle can be controlled using the ``SRx_*`` parameters (i.e. :ref:`SR1_POSITION <SR1_POSITION>`).
 
+SRx are not connected directly to Serial port number but follow number of mavlink enabled. So, for example, if Mavlink is enabled on Serial 1, Serial 2 and Serial 4, than SR1 will refer to Mavlink on Serial 1, SR2 will refer to Mavlink on Serial 2 and SR3 will refer to Mavlink enabled on Serial 4.
```

Comment:
Perhaps a little clearer wording would be: "SRx parameters do not directly correlate to the SERIALx port, but rather to the instance ordering of ports using Mavlink."

Reply:
Ok, how do i change it?

Reply:
I wrote it to be understandable to people to wich words like instance are not so familiar :)

Reply:
the wording as is , is not clear English..."are not connected"... and "but follow the number of MAVLink enabled...".....the verbs are less clear than "associate" or correlate" or "correspond"  instead of connect or follow

here is another version: "SRx parameters do not directly correlate to the SERIALx port, but rather to the order of the ports which have MAVLink protocol active."

you change it by modifying your patch-1 branch , squashing back to a single commit, force push to your origin....that will automatically be reflected here....OR, I can do it for you and then merge it...whichever you want

Reply:
I would like to learn how to do it but i am not sure i understand how

Reply:
I tried, don't know if it is ok now.

Reply:
Have 2 commits now, probably did something wrong......

Reply:
not a problem...you probably made the contribution directly from the wiki page, instead of using a local git repository....if you are not familiar with GIT commands, setting up GIT on your PC and creating a local copy of a repo,etc. its the easiest way.....but you cannot do what I had suggested about squashing the commits....dont worry....it looks good now....I can squash them and then merge it....thanks for the contribution! soon you will see that force pushed changes to your github ardupilot_wiki repository making it a single commit (avoids cluttering  up the history list of commits) and then merging the PR into the master branch of the wiki...